packages/contracts/contracts/Proxy/BorrowerWrappersScript.sol

```diff
@@ -138,6 +138,9 @@ contract BorrowerWrappersScript is BorrowerOperationsScript, ETHTransferScript,
     }
 
     function _requireUserHasTrove(address _depositor) internal view {
-        require(troveManager.getTroveStatus(_depositor) == 1, "BorrowerWrappersScript: caller must have an active trove");
+        require(
+            troveManager.getTroveStatus(_depositor) == ITroveManager.Status.active, 
```

Comment:
Admittedly a nitpick. Should TroveManager have this common function; I think it was also used in `BorrowerOperations` and some other spots? Eg.
```
troveManager.isTroveActive(...)
```

Reply:
I tried to touch as less as possible of Liquity code, so I'll have a look in case of more changes in that part